mm/2s2h/Enhancements/Cutscenes/SkipEntranceCutscenes.cpp

```diff
@@ -1,6 +1,7 @@
-#include <libultraship/libultraship.h>
+#include "public/bridge/consolevariablebridge.h"
 #include "2s2h/GameInteractor/GameInteractor.h"
 #include "2s2h/ShipInit.hpp"
+#include "variables.h"
```

Comment:
Here and a couple of these other enhancement files where you added `variables.h`, could you place them inside `extern "C" { }` blocks if they aren't already, to match our existing patterns
```suggestion
#include "2s2h/ShipInit.hpp"

extern "C" {
#include "variables.h"
}
```